README.md

```diff
@@ -52,6 +52,7 @@ LEDGER_ACCOUNT="<your Ledger deployer address>"
 DEPLOY_STRATEGY=create2
 
 STATE_CONTRACT_ADDRESS=
+SMT_LIB_ADDRESS=
```

Comment:
Is this `SMT_LIB_ADDRESS` correct? I don't see this env variable used in the deployment/upgrade scripts like the other env variables.